### PR TITLE
post gen modal as design

### PR DESCRIPTION
**Pull request by Tanya0609**
Based on the user testing, the current post generation modal is confusing to use. Thus, Christina came up with a new design. This PR is to make post generation functions as the new design. 

**How to test:**
1. try to generate a project with or without Azure Services 
2. when click **create project**, the post generation modal should look as expected

**Expected ( with Azure Services selected):**
post-modal

### CODE REVIEW

**Comment by markAtMicrosoft** [comment]
White border seems to be thinner on bottom.  Screen clip attached.  This was on a 1920x1080 device.  showed normal on my SB screen of 3000x2000
image

**Comment by markAtMicrosoft** [comment]
The "Open README.md to start" might be clearer as "After 'Open Project' see README.md to start"

**Comment by markAtMicrosoft** [comment]
> White border seems to be thinner on bottom.  Screen clip attached.  This was on a 1920x1080 device.  showed normal on my SB screen of 3000x2000

File: src/client/src/containers/PostGenerationModal/strings.ts [hunk 1]
```diff
@@ -51,19 +51,35 @@ export const strings = defineMessages({
...
+    defaultMessage: "Open "
+  },
+  readMe: {
+    id: "postGenerationModal.readme",
+    defaultMessage: "README.md"
+  },
+  toStart: {
+    id: "postGenerationModal.toStart",
+    defaultMessage: " to start"
   },
```
Comment (markAtMicrosoft) [inline src/client/src/containers/PostGenerationModal/strings.ts:83]: Would this be better as **"After 'Open Project' see README.md to start"**

Comment (Tanya0609) [reply src/client/src/containers/PostGenerationModal/strings.ts:83]: PR updated :)

**Review by piotrmark — APPROVED** [review]

**Review by SahilTara — APPROVED** [review]

**Comment by jasmingoh** [comment]
Post generation modal looks good!

Just a tiny concern - when I click on "Create New Project", I was directed to the first page of the wizard with an error showing for the output path.

image

Also, I have a default output path set, but it seems like it wasn't showing after clicking on "Create New Project"

**Comment by Tanya0609** [comment]
@jasmingoh I did not touch the Create New Project logic, it is the same as Restart Wizard. This may introduce due to multiple UI changes. I will take a look at this issue and will be handled in a different PR.

**Review by jasmingoh — APPROVED** [review]
Looks good! 

GitHub issue created for my comment above, #872

**Pull request merged**